### PR TITLE
Fix at Tag.Informative

### PR DESCRIPTION
**Pull request by ericcleao**
## Changes:

- Set variant props as optional;
- Fix the color used at the default variant;
- Start to use the `Icon` component at the web `Tag`;
- Refactor some details using the system that is integrated at the `Icon`.

## Doc

image (1)

### CODE REVIEW

File: packages/yoga/src/Tag/native/Informative.jsx [hunk 1]
```diff
@@ -81,14 +82,15 @@ const TagInformative = ({
 
 TagInformative.propTypes = {
   /** style the tag following the theme (success, informative, attention) */
-  variant: oneOf(['success', 'informative', 'attention']).isRequired,
+  variant: oneOf(['', 'success', 'informative', 'attention']),
```
Comment (coelhucas) [inline packages/yoga/src/Tag/native/Informative.jsx:85]: What do you think about having a `default` variant instead?

```suggestion
  variant: oneOf(['default', 'success', 'informative', 'attention']),
```
Then below
```diff
TagInformative.defaultProps = {
--  variant: '',
++  variant: 'default',
```
Same suggestion for web

**Review by coelhucas — COMMENTED** [review]
I have mixed feelings when it comes to using the logical and to conditional renderings, I'm leaning towards ternaries instead, but you can also convert icon prop to boolean with `!!icon`. Up to you, this isn't necessarily a change request.

[Reference on this topic](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx).

**Comment by ggdaltoso** [comment]
We have some misconceptions between our designers and it would be better to hold this PR closed until they decide what they want to do within this default Tag

**Comment by ggdaltoso** [comment]
@ericcleao 

There is some good things on this PR we could take advantage of.

What do you think about redoing the changes adding the gray Tag and just ship the refactors?
In addition to that, we could add a better default prop to Tag, preventing of been use wrongly.

**Comment by ericcleao** [comment]
@ggdaltoso I believe it's easier if we just close this PR and use the interesting part that involves the Icon at the Tag component at other PR, and afterward, we update the tag default color if it's needed.

**Comment by ericcleao** [comment]
New PR updating the icon approach at the Tag: https://github.com/Gympass/yoga/pull/355

**Pull request closed without merging**